aviary/utils/process_input_decks.py

```diff
@@ -139,6 +139,11 @@ def parse_inputs(vehicle_deck, aircraft_values: AviaryValues(), meta_data=_MetaD
                 initial_guesses[var_name] = float(var_values[0])
                 continue
 
+            else:
+                warnings.warn(
+                    f"Variable '{var_name}' is not in meta_data nor in 'guess_names'. It is ignored.")
+                continue
+
             if aircraft_values.get_val('verbosity').value >= 2:
                 print('Unused:', var_name, var_values, comment)
```

Comment:
we originally decided to use a print instead of a warning because there are many unused variables we don't want warnings for. Maybe we could change this to be a warning if the variable name contains a ":"

Reply:
or if the variable name starts with "aircraft" or "mission"

Reply:
I use ":" to do the test.

Reply:
@crecine

I guess you mean the sections of unconverted variables that are placed at the bottom of the csv file by the converter?  I'm not sure we should design around that. Maybe we should comment out those lines instead.